app/main.py

```diff
@@ -1,8 +1,43 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+
+    def __init__(self, distance_from_city_center: float,
+                 clean_power: int, average_rating: float,
+                 count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars_list: list) -> float:
+        total_price = 0
+        for car in cars_list:
+            if car.clean_mark < self.clean_power:
+                income_from_washing_single_car = \
+                    self.calculate_washing_price(car)
+                total_price += income_from_washing_single_car
+                self.wash_single_car(car)
+        return total_price
+
+    def calculate_washing_price(self, instance: Car) -> float:
```

Comment:
```suggestion
    def calculate_washing_price(self, car: Car) -> float:
```

`instance` isn't a descriptive name